core-bundle/contao/templates/twig/content_element/code.html.twig

```diff
@@ -8,23 +8,21 @@
         {% block editor_view %}<pre>{{ code }}</pre>{% endblock %}
     {% else %}
         {% block highlighted %}
-            {% block style %}
-                {% add "highlighter_css" to head %}
-                    {% with {file: highlighter_css_file} %}{{ block('stylesheet_component') }}{% endwith %}
-                {% endadd %}
-            {% endblock %}
-
-            {% block code %}
```

Comment:
Did you delete the block `code` on purpose?
This would break all child templates that extended that block.